README.md

```diff
@@ -226,14 +244,19 @@ control "operating_system" do
 end
 ```
 
-Running the following command would initialize the working directory for
-Terraform, create a Docker container by applying the configuration file,
-and verify that the container is running Ubuntu.
+Running the following commands would install Kitchen-Terraform,
+initialize the working directory for Terraform, create a Docker
+container by applying the configuration file, and verify that the
+container is running Ubuntu.
 
 > Verifying with Kitchen-Terraform
 
```

Comment:
I feel like there should be a sentence between the title and the example.  Maybe something like:

The following example uses bundler to run kitchen-terraform as describe here (link to bundler stuff above). Test kitchen runs when using bundler should use the following pattern:
```
bundle exec kitchen <command>
```
For example: